FACBackEnd/src/ingestData.js

```diff
@@ -0,0 +1,22 @@
+const csv = require('csv-parser')
+const fs = require('fs')
+const putItem = require('./putItem')
+
+function readFromCSV (relativePath) {
+  fs.createReadStream('./sampledata/seattlecanlocations.csv')
```

Comment:
Hardcoded file path is not good. Should likely be relativePath. Also unclear what the format of the .csv needs to be

Reply:
Add comment on the form of what the csv should be in